src/MapKit/MKEnums.cs

```diff
@@ -140,6 +143,24 @@ public enum MKFeatureVisibility : long {
 		Hidden,
 		Visible,
 	}
+
+	[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+	[Native]
+	public enum MKLocalSearchCompleterResultType : long
+	{
+		Address = 1 << 0,
+		PointOfInterest = 1 << 1,
+		Query = 1 << 2,
+	}
+
+	[TV (13,0), NoWatch, Mac (10,15), iOS (13,0)]
+	[Native]
+	public enum MKLocalSearchResultType : ulong
```

Comment:
same for `[Flags]`
sharpie does it (but not always) depending on how it's declared in ObjC